source/shared/core_conn.cpp

```diff
@@ -278,13 +445,6 @@ SQLRETURN core_odbc_connect( _Inout_ sqlsrv_conn* conn, _Inout_ std::string& con
     memset( wconn_string, 0, wconn_len * sizeof( SQLWCHAR )); // wconn_len is the number of characters, not bytes
```

Comment:
memset_s

Reply:
actually if there is a concern with sensitive data, then memset should definitely not be used as it's going to get optimized out.  I'm not sure if the compiler won't do the same with memset_s too? Depending on the platform I think you would need to cast it to a volatile pointer or pragma disable optimization here? hmm need to check tomorrow to see how ODBC handles this